app/views/layouts/application.html.haml

```diff
@@ -47,6 +47,10 @@
   = succeed '.' do
     = link_to "Find out more about cookies", cookies_path
 
+- content_for :after_header do
```

Comment:
shouldn't we just put this in: https://github.com/ministryofjustice/prison-visits/blob/master/app/views/deferred/visits/show.html.haml